### ISSUE TITLE
Support for string types for "show_in_menu"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#185]
```
What functionality do you want to see?

See this post: 

http://wordpress.org/support/topic/how-do-i-rename-post-type?replies=3#post-2363
602

Basically, you can supply a string for this value and use it to put your custom 
post type menus inside other menus (i.e. hierarchical menus).  This might run 
into the menu options that were implemented in 0.9.4 (see issue 178).
```

Original issue reported on code.google.com by `fireproofsocks` on 28 Sep 2011 at 3:09

**Comment by GoogleCodeExporter** [comment]
```
Jup, this would be a great addition!
```

Original comment by `toineka...@gmail.com` on 28 Sep 2011 at 6:41

**Comment by GoogleCodeExporter** [comment]
```
This has been added and will be released in 0.9.4.
```

Original comment by `fireproofsocks` on 3 Oct 2011 at 5:31
- Changed state: **Fixed**
